### ISSUE TITLE
Reddit: This video does not have audio

### ISSUE THREAD

**Issue by zoli456** [DuncteBot/skybot-lavalink-plugin#21]
```
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: This video does not have audio
        at com.dunctebot.sourcemanagers.reddit.RedditAudioSourceManager.buildTrack(RedditAudioSourceManager.java:196) ~[skybot-lavalink-plugin-1.4.2.jar!/:na]
        at com.dunctebot.sourcemanagers.reddit.RedditAudioSourceManager.loadItem(RedditAudioSourceManager.java:74) ~[skybot-lavalink-plugin-1.4.2.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:449) ~[lavaplayer-1.4.2.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:431) ~[lavaplayer-1.4.2.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:223) ~[lavaplayer-1.4.2.jar!/:na]
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
        at java.base/java.lang.Thread.run(Thread.java:830) ~[na:na]
        Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation: 
  lavaplayer.version: 1.4.2-original
  os.arch: amd64
  os.name: Linux
  os.version: 5.4.0-146-generic
  java.vendor: Oracle Corporation
  java.version: 13.0.2
  java.runtime.version: 13.0.2+8
  java.vm.version: 13.0.2+8
``` 
exp.: https://www.reddit.com/r/funnyvideos/comments/15cvyaj/cant_find_this_woman_but_she_is_smart/
I tried this video but it say it havent got audio but video does have audio. I can play it directly.

**Comment by duncte123** [comment]
🙃 It's the most wonderful time of the year

**Comment by duncte123** [comment]
Fixed in version `1.6.3`, please keep in mind that this is on a new maven repository: `https://maven.lavalink.dev/releases`
